shared/js/media/crop_resize_rotate.js

```diff
@@ -72,14 +74,13 @@
 //    shared/js/media/image_size.js
 //    shared/js/media/jpeg_metadata_parser.js
 //    shared/js/media/downsample.js
+//    shared/js/media/jpeg-exif.js (with LazyLoader)
 //
 function cropResizeRotate(blob, cropRegion, outputSize, outputType,
                           metadata, callback)
 {
-  'use strict';
-
-  const JPEG = 'image/jpeg';
-  const PNG = 'image/png';
+  const JPEGType = 'image/jpeg';
+  const PNGType = 'image/png';
```

Comment:
I'd rather alter jpeg-exif.js so that it defined something with a clearer name then JPEG.  Then you could leave this constant unmodified.   Maybe call it JPEGParser or EXIFParser instead of just plain JPEG